src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,128 @@
+* Add support for node public IP prefix ID (#17138)
+
+**APIM**
+
+* [BREAKING CHANGE] `apim backup`: `--storage-account-container` not support multi-value. (#17315)
+* [BREAKING CHANGE] `apim restore`: `--storage-account-container` not support multi-value. (#17315)
+
+**App Service**
+
+* [BREAKING CHANGE] Fixes #16087:az webapp config ssl create  setting --name parameter as required. (#17079)
+* Fix #17053: `az webapp show` returning null values for SiteConfig properties (#17054)
+* Fix #17207: for az webapp log config 'level' always defaults to verbose (#17259)
+
+**ARM**
+
+* `az bicep build`: fix an issue where build warnings are not shown (#17180)
+
+**Backup**
+
+* Add `id_part` for sub-resource names to fix `--ids` (#17165)
+* Fix #17094: Created separate test suite for CRR tests (#17183)
+* `az backup protection check-vm`: Add vm and resource-group as optional params (#16974)
+
+**Cache**
+
+* GA `az cache` (#17264)
+
+**CDN**
+
+* `az afd rule create`: Fix '--help' message (#17282)
+
+**Compute**
+
+* Fix a Windows vm user update bug (#17257)
+* Fix #16585: az vmss deallocate --instance-ids failed (#17274)
+* `az vm create`: New parameter --platform-fault-domain in FLEX VMSS mode (#16409)
+* `az vm create`: --patch-mode for Linux VM (#16409)
+* `az ssh vm`: Automatically launch browser when getting certificate fails (#17093)
+* `az vm create`: New parameter --count (#17217)
+* `az vm create`: Trusted Launch (#17354)
+
+**Extension**
+
+* Add actionable message when an extension is not compatible with the CLI core (#16751)
+
+**Key Vault**
+
+* `az keyvault role definition list`: Support `--custom-role-only` to list only custom role definitions (#17119)
+* Support keyvault custom role definition (#17109)
+* Add `--no-wait` for command `az keyvault security-domain download` and `--target-operation` for command `az keyvault security-domain wait` (#17263)
+
+**NetAppFiles**
+
+* `az netappfiles account backup show`: Operation added. (#17173)
+* `az netappfiles account backup delete`: Operation added. (#17173)
+* `az netappfiles account ad add`: Parameter ldap-over-tls added. (#17173)
+* `az netappfiles account create`: Parameter encryption added. (#17173)
+* `az netappfiles account update`: Parameter encryption added. (#17173)
+* `az netappfiles volume create`: Parameter encryption-key-source added. (#17173)
+* `az netappfiles volume create`: Default export policy removed for nfsv4.1 and optional parameters added for setting up an export policy for nfsv4.1: rule_index, unix_read_only, unix_read_write, cifs, allowed_clients (#17173)
+
+**Network**
+
+* `az network public-ip prefix  create`: Support `--zone 1 2 3` (#17279)
+* `az network lb  frontend-ip create`: Support `--zone 1 2 3` (#17279)
+* Bump version from '2020-08-01' to '2020-11-01' (#17290)
+* `az network lb address-pool` : Support subnet when creating or updating an IP-based backend pool of a load balancer. (#17336)
+
+**Packaging**
+
+* Add azure-identity dependency (#17277)
```

Comment:
Move to core?

Reply:
`azure-identity` is required by RDBMS, so belongs to `azure-cli`, not `azure-cli-core`.